manticore/platforms/linux.py

```diff
@@ -1412,7 +1412,11 @@ def sys_arch_prctl(self, code, addr):
         ARCH_SET_FS = 0x1002
         ARCH_GET_FS = 0x1003
         ARCH_GET_GS = 0x1004
-        assert code == ARCH_SET_FS
+        if code not in (ARCH_SET_GS, ARCH_SET_FS, ARCH_GET_FS, ARCH_GET_GS):
+            logger.debug("code not in expected options ARCH_GET/SET_FS/GS")
+            return -errno.EINVAL 
```

Comment:
```suggestion
            return -errno.EINVAL
```
(remove trailing whitespace)